dist/isoCurrency.js

```diff
@@ -894,8 +894,16 @@ angular.module('isoCurrency', ['isoCurrency.common'])
 			if (!currency) {
 				return amount;			
 			}
+
 			var fractionSize = (fraction === void 0) ? currency.fraction : fraction;
-			return $filter('currency')(amount, currency.symbol, fractionSize);
+	        var currencyFilter = $filter('currency');         
+
+	        if(amount < 0){
```

Comment:
@makdad Okay, I just wanted to hear your reasoning even though it was trivial to add the comment. I appreciate the explanation.